test/scripts/e2e-kind.sh

```diff
@@ -45,7 +48,42 @@ should set default value on new IngressClass
 should prevent Ingress creation if more than 1 IngressClass marked as default
 "
 
-SKIPPED_TESTS=$(echo "${SKIPPED_TESTS}" | sed -e '/^\($\|#\)/d' -e 's/ /\\s/g' | tr '\n' '|' | sed -e 's/|$//')
+IPV4_ONLY_TESTS="
+# Limit the IPv4 related test to IPv4 only deployments
+#  See: https://github.com/leblancd/kube-v6-test
+\[Feature:Networking-IPv4\]
+
+# The following tests currently fail for IPv6 only, but should be passing.
+# They will be removed as they are resolved.
+
+# shard-n Tests
+Network.+should resolve connrection reset issue
+
+# shard-np Tests
+NetworkPolicy.+should allow egress access to server in CIDR block
+
+# shard-s Tests
+Services.+should create endpoints for unready pods
+
+# shard-other Tests
```

Comment:
we should stop sharding tests, is not giving a good signal of the project health, the goal  is not only to just pass the tests, is to pass them unders stress :smile: 
I have an example of this, running all the network tests in parallel in the same job just take 11 mins only
https://github.com/aojea/ovn-kubernetes/actions/runs/167491411